config/localization/en/app.json

```diff
@@ -2061,22 +2061,23 @@
     "V4_TESTNET_IMPERATOR": "V4 Testnet 3(Imperator)"
   },
   "ERRORS": {
-    "BROADCAST_ERROR_2000": "Fill Or Kill order could not be fully filled.",
-    "BROADCAST_ERROR_2001": "Reduce-only orders cannot increase the position size.",
-    "BROADCAST_ERROR_2002": "Reduce-only orders cannot change the position side.",
-    "BROADCAST_ERROR_2003": "Post-only order would cross one or more maker orders.",
-    "BROADCAST_ERROR_2005": "Order would violate isolated subaccount constraints.",
-    "BROADCAST_ERROR_3000": "Invalid order flags.",
-    "BROADCAST_ERROR_3001": "Invalid order goodTilBlockTime.",
-    "BROADCAST_ERROR_3002": "Stateful orders cannot require immediate execution.",
-    "BROADCAST_ERROR_3003": "The block time is greater than the GoodTilBlockTime of the message.",
-    "BROADCAST_ERROR_3004": "The GoodTilBlockTime of the message is further than StatefulOrderTimeWindow into the future.",
-    "BROADCAST_ERROR_3005": "Existing stateful order has higher-or-equal priority than the new one.",
-    "BROADCAST_ERROR_3006": "Stateful order does not exist.",
-    "BROADCAST_ERROR_3007": "Stateful order collateralization check failed.",
-    "BROADCAST_ERROR_3008": "Stateful order was previously cancelled and therefore cannot be placed.",
-    "BROADCAST_ERROR_3009": "Stateful order was previously removed and therefore cannot be placed.",
-    "BROADCAST_ERROR_10001": "Subaccount cannot open more orders due to equity tier limit. {EQUITY_TIER_LEARN_MORE}",
```

Comment:
Can we preserve the older BROADCAST errors so that mobile does not lose strings if they do not have a build with the updated Abacus changes?

Reply:
ooo actually will older mobile use an older localization?

Reply:
I think they pull directly from [dydx-v4-shared-resources.vercel.app](https://dydx-v4-shared-resources.vercel.app/)